types/redux-storage/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "private": true,
     "dependencies": {
-        "redux": "^3.6.0"
+        "redux": "^3.6.0 || ^4.0.0"
```

Comment:
Is there a reason not to just take 4.0.0? I'm assuming this condition does the same thing.

Reply:
They actually do different things: `^3.6.0` mean "3.x.x" and `^4.0.0` mean `4.x.x` - please check at https://semver.npmjs.com/